SlackChatbot/slack_bot.py

```diff
@@ -0,0 +1,107 @@
+from flask import Flask, request, jsonify
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+from trello import TrelloClient
+import threading
+import time
+import os
+import re
+
+# Initialize Slack and Trello clients
+client = WebClient(token='xoxb-7855184757495-7855216149575-KI6lOoBXKSoudrc9pzugoW0x')
```

Comment:
It is bad practice to hard-code your tokens. You should consider using `os.getenv()` as you have done below